### ISSUE TITLE
No table of content in jupyter lab 3.0.7

### ISSUE THREAD

**Issue by fcollonval** [jupyterlab/jupyterlab-toc#167]
Hello, I just installed jupyter lab 3.0.7 and I don't see the TOC anywhere. Here is what I see:

![Screenshot 2021-04-12 at 11 58 40](https://user-images.githubusercontent.com/45604439/114384364-86894c80-9b86-11eb-8947-2cc3a5b0fcf6.png)

Is there something that I can do to fix this?

_Originally posted by @DonovanAD in https://github.com/jupyterlab/jupyterlab-toc/issues/166#issuecomment-817714126_

**Comment by fcollonval** [comment]
@DonovanAD could you try reloading the page?

If it still does not show up, could you report the output of the command `jupyter labextension list`.

**Comment by DonovanAD** [comment]
Hello @fcollonval, I tried reloading the page but nothing changed. I also tried uninstalling and installing multiple times using `pip` and `conda`, but it never worked. 

However, I managed to solved it already. I had to follow what `krassowski ` suggested [here](https://stackoverflow.com/questions/65952807/uninstall-jupyter-lab-with-conda-completely) and I removed all folders that I could have accessed to when typing `jupyter --paths`:


config:

    /Users/USERNAME/.jupyter ------------------> REMOVED BY ME
    /Users/USERNAME/anaconda3/etc/jupyter -----> REMOVED BY ME
    /usr/local/etc/jupyter
    /etc/jupyter

data:

    /Users/USERNAME/Library/Jupyter -----------> REMOVED BY ME
    /Users/USERNAME/anaconda3/share/jupyter ---> REMOVED BY ME
    /usr/local/share/jupyter
    /usr/share/jupyter

runtime:

    /Users/USERNAME/Library/Jupyter/runtime ---> REMOVED BY ME


and then uninstall everything showed when typing `jupyter --version` in the terminal using `pip`:

> jupyter core     
jupyter-notebook 
qtconsole       
ipython          
ipykernel        
jupyter client   
jupyter lab      
nbconvert        
ipywidgets       
nbformat       
traitlets   

     

Then I proceeded to reinstall using `pip install jupyter` and `conda install -c conda-forge jupyterlab=3` and now everything is working as it should be.

I experienced the problem with TOC not showing that I described on:

- macOS Big Sur version 11.2.3 (20D91)
- chrome Version 89.0.4389.114 (Official Build) (x86_64)
- jupyter lab 3.0.7

And currently everything is working with:

- same OS and browser 
- juyter lab 3.0.13
